.github/workflows/release.yml

```diff
@@ -31,27 +32,28 @@ jobs:
         uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
         with:
           fetch-depth: 0
+          persist-credentials: false
```

Comment:
Open question: does this need to be added? I'm personally fine with its default unless it mucks with the `git push` -- similar for below

Reply:
Good question, it does make the push cleaner since `action/checkout` adds a header `http.https://github.com/.extraheader` to all operations that takes priority over the remote URL. So before our push we would have to run the same commands that this `persist-credentials` runs, example (probably just the unset command seen below):

![image](https://user-images.githubusercontent.com/8650838/219680231-3400f0a2-6fea-440f-aa28-bc3984c00e58.png)

More [detail](https://github.com/actions/checkout/blob/b4626ce19ce1106186ddf9bb20e706842f11a7c3/adrs/0153-checkout-v2.md#persist-credentials)

Reply:
Sheesh that's unfortunate we have to workaround that. Helpful information. 👍 

Would you mind dropping a quick code comment about this behavior near this non-default? I think that could help future travelers understand why its there or figure out if its safe to remove in the future should things change.

Reply:
Absolutely, added ⚡